### DISCUSSION TITLE
Tailwind v3 installation in laravel

### DISCUSSION THREAD

**Discussion by MiraahA380** [tailwindlabs/tailwindcss#6554] [Help]
I have tried installing tailwind using Laravel breeze and it seems that only a very few utility classes are available. It seems like it is purged by default. How to solve this issue?

**Comment by manhthang2504** [comment]
Yes, from Tailwind 3, it perform JIT compile all the time. It mean `npm run dev` is now the same as `npm run prod `

**Comment by cdotd** [comment]
It seems i have the same problem but without breeze.
I did this steps: https://tailwindcss.com/docs/guides/laravel

The problem i see is in the css file:
[https://pastebin.com/6K8SWMaT](url)
but there are no errors...

Someone has an idea?

**Reply by MiraahA380** [reply]
Hey guys! I just found out there's nothing wrong here. Its just that Tailwind has just released its v3.0 . You can find about it here....https://tailwindcss.com/docs/upgrade-guide

**Comment by lukewarlow** [comment]
Have you correctly configured the "content" section of your config. See https://tailwindcss.com/docs/content-configuration and https://tailwindcss.com/docs/upgrade-guide#configure-content-sources. Is there a repository that you can link to that someone could take a look at?
